src/index.html

```diff
@@ -5,8 +5,347 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.gstatic.com"><link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700& display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a class="header__logo" href="#">
+        <img src="./images/logo.png" alt="Moyo picture">
+      </a>
+      <nav class="nav">
+        <ul class="nav_items">
+          <li class="nav_item">
+            <a href="#" class="item_link is-active">APPLE
+            </a>
+          </li>
+          <li class="nav_item">
+            <a href="#" class="item_link">SAMSUNG
```

Comment:
```suggestion
            <a href="#" class="item__link">SAMSUNG
```
Элементы блока лучше писать через __